### PR TITLE
Description for -RemoveCanceledMeetings property

### PR DESCRIPTION
**Pull request by sama-zaki**
Updated Set-CalendarProcessing.md with a description for -RemoveCanceledMeetings property that was missing.

### CODE REVIEW

**Comment by sama-zaki** [comment]
@chrisda Here is the PR for the property I added a description to.

File: exchange/exchange-ps/exchange/Set-CalendarProcessing.md [hunk 1]
```diff
@@ -796,9 +796,10 @@ Accept wildcard characters: False
 ```
 
 ### -RemoveCanceledMeetings
-This parameter is available only in the cloud-based service.
+This parameter is available only in the cloud-based service. The RemoveCanceledMeetings parameter specifies whether to automatically delete  meetings that were canceled by organizer from the resource mailbox's calendar.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-CalendarProcessing.md:799]:
```suggestion
This parameter is available only in the cloud-based service. The RemoveCanceledMeetings parameter specifies whether to automatically delete meetings that were canceled by the organizer from the resource mailbox's calendar.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sama-zaki, kindly commit the minor suggested change. Many thanks.

**Comment by sama-zaki** [comment]
Done @JohanFreelancer9

**Pull request merged**